scripts/merge-forward.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Forward merge 'master' branch into 'v2-main' branch
+
+set -exo pipefail
+
+git checkout -b v2-main origin/v2-main
```

Comment:
Maybe worth making the branch name unique?

Shouldn't we precede this with a `git fetch`?

Reply:
The command that precedes this is a `git clone` - https://github.com/awslabs/aws-delivlib/blob/main/lib/pull-request/pr.ts#L321 - so a fetch shouldn't be needed. 

> Maybe worth making the branch name unique?

This is only the local branch during this build. I'll make the remote PR branch unique.